utils/navigator_gui/navigator_gui/host_monitor.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+
+'''
+Host Monitor: A simple node to monitor and publish the IP address status of the
+core hosts on the platform's network.
+'''
+
+
+import socket
+import subprocess
+
+from navigator_msgs.msg import Hosts
+import rospy
+
+
+__author__ = "Anthony Olive"
+__maintainer__ = "Anthony Olive"
+__email__ = "anthony@iris-systems.net"
+__copyright__ = "Copyright 2016, MIL"
+__license__ = "MIT"
+
+
+rospy.init_node("host_monitor")
+
+
+class HostMonitor():
+
+    def __init__(self):
+        self.pub_hosts = rospy.Publisher("/host_monitor", Hosts, queue_size=1)
+
+        # Build an ordered list of host dictionaries that resolve to devices on navigator
+        host_list = [
+            "mil-nav-ubnt-wamv",
```

Comment:
mil-nav-ubnt-wamv = The Ubiquity on NaviGator
mil-nav-ubnt-shore = The Ubiquity on shore
mil-nav-wamv = The primary computer
mil-nav-perception = The secondary computer
mil-com-velodyne-vlp16 = The Velodyne LiDAR
mil-com-sick-lms111 = The Sick LiDAR